terraform/performance/main.tf

```diff
@@ -45,13 +45,14 @@ module "validator" {
 #####################################################################
 resource "aws_instance" "cwagent" {
 
-  ami                         = data.aws_ami.latest.id
-  instance_type               = var.ec2_instance_type
-  key_name                    = local.ssh_key_name
-  iam_instance_profile        = module.basic_components.instance_profile
-  vpc_security_group_ids      = [module.basic_components.security_group]
-  get_password_data           = local.connection_type == "winrm" ? true : false
-  associate_public_ip_address = true
+  ami                                  = data.aws_ami.latest.id
+  instance_type                        = var.ec2_instance_type
+  key_name                             = local.ssh_key_name
+  iam_instance_profile                 = module.basic_components.instance_profile
+  vpc_security_group_ids               = [module.basic_components.security_group]
+  get_password_data                    = local.connection_type == "winrm" ? true : false
```

Comment:
nitpick: can this line be simplified to local.connection_type == "winrm"?

Reply:
Not sure, it was an existing line, the `make` target just auto formatted the whitespace.
I would prefer to leave it.

Reply:
https://developer.hashicorp.com/terraform/language/expressions/conditionals